### PR TITLE
Added blog for Mayastor release

### PR DESCRIPTION
**Pull request by satyapriyamishra222**
Signed-off-by: satyapriyamishra222 <satyapriyamishra222@gmail.com>

### CODE REVIEW

File: website/src/blogs/mayastor-1.0-has-arrived.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+Built on the Container Attached Storage framework, the storage engine leverages a hyper-converged deployment model, where data is stored locally then shared with other nodes via synchronous replication.
+
+## Mayastor Architecture 
+
+The platform is built for cloud-native storage orchestration and is workload-driven to simplify Kubernetes storage for enterprises. Through a declarative data plane, Mayastor abstracts storage resources to enable persistent storage for Kubernetes applications. To do so, Mayastor storage engine components are deployed as containers in Kubernetes, allowing for simple scaling, provisioning and management of storage for clusters.
+
+**Control Plane -**The Mayastor control plane is primarily a single instance Kubernetes controller that implements both the Container Storage Interface (CSI) specification and private interfaces for the storage system. The controller is containerized and runs as deployment in Kubernetes. The control plane also includes CSI plugins deployed on each node to implement CSI protocol services.
```
Comment (Pallavi-PH) [inline website/src/blogs/mayastor-1.0-has-arrived.md:24]:
```suggestion
**Control Plane -** The Mayastor control plane is primarily a single instance Kubernetes controller that implements both the Container Storage Interface (CSI) specification and private interfaces for the storage system. The controller is containerized and runs as deployment in Kubernetes. The control plane also includes CSI plugins deployed on each node to implement CSI protocol services. 
```

**Comment by Pallavi-PH** [comment]
Can we optimize the images, please?

File: website/src/blogs/mayastor-1.0-has-arrived.md (hunk 1 above)
Comment (satyapriyamishra222) [reply website/src/blogs/mayastor-1.0-has-arrived.md]: This change is done.

**Comment by satyapriyamishra222** [comment]
> Can we optimize the images, please?

Done

**Review by Prasoon1993 — APPROVED** [review]

**Pull request merged**